### ISSUE TITLE
Get-GivenName using WMI returns the last name when using 'Last, First' formatting

### ISSUE THREAD

**Issue by bbrownchg** [imabdk/Toast-Notification-Script#3]
My organization uses 'Last, First' for our display names, so when the Get-GivenName function runs and has to retrieve the information from WMI the last name is returned instead of the first name. The specific command is on line 262:

`$GivenName = $DisplayName.Split()[0].Trim()`

I can easily remedy that by changing the split to 1:

`$GivenName = $DisplayName.Split()[1].Trim()`

Not sure if you want to document this somewhere, or find a way to account for differences in formatting between organizations.

**Comment by imabdk** [comment]
Right. I haven't thought about that. I'll see what I can do to cater for that situation. Thanks for making me aware. Much appreciated! :-)

**Comment by mjelic91** [comment]
Maybe implementing it to be used from custom XMLs? :)
